core/trino-main/src/main/java/io/trino/execution/StageStats.java

```diff
@@ -626,6 +626,14 @@ public List<OperatorStats> getOperatorSummaries()
         return operatorSummaries;
     }
 
+    @JsonProperty
+    public DataSize getSpilledDataSize()
```

Comment:
Could that be computed in JS instead? Sending it is redundant to JSON structure.